mesonbuild/optinterpreter.py

```diff
@@ -77,9 +77,6 @@ def __init__(self, subproject, command_line_options):
             if self.subproject != '': # Strip the beginning.
                 if not o.startswith(self.sbprefix):
                     continue
-            else:
-                if ':' in o:
-                    continue
```

Comment:
The reason it is there is that the full option name is `subproject:optionname`. Colon is reserved as a separator, otherwise there would be a clash between toplevel option `foo:bob` and option `bob` in subproject `foo`.